### DISCUSSION TITLE
Staticman in jekyll leads to 405 “not allowed”

### DISCUSSION THREAD

**Discussion by cmeissner** [eduardoboucas/staticman#425] [General]
I tried to configure 'staticman' comments for my github pages. I use Beautiful Jekyll for my site.

I followed the documentation of [staticman getting-started](https://staticman.net/docs/getting-started).

1. created a github application and install it in my [repo](https://github.com/codeaffen/codeaffen.org)
2. deploy staticman app on heroku
3. configure `_config.yml`
4. configure `staticman.yml`

Unfortunately I get only a HTTP 405 (not allowd) when I try to submit a comment. There does not arrive any request at my staticman installation. So I would guess this is a problem at github pages side.

**Comment by cmeissner** [comment]
Obviously the post request is never send to my static man instance but to the post site itself. As github pages does not allow requests other than GET the 405 is the correct answer. So now I assume that there is a problem around the jquery magic.

**Comment by cmeissner** [comment]
Seems to be related to [beautiful-jekyll#766](https://github.com/daattali/beautiful-jekyll/issues/766)

**Reply by VincentTam** [reply]
> Seems to be related to [beautiful-jekyll#766](https://github.com/daattali/beautiful-jekyll/issues/766)

Hi, I've ported Staticman from another Jekyll theme to your theme.  Points 3 & 4 are front-end setup, so IMHO, the theme's repo is a better place for these issues.  This repo is about the back-end setup.

I've resolved the linked issue by PR daattali/beautiful-jekyll<span>#</span>782, thanks to the Hugo Swift theme's author.
